ui/ui-app/src/main/resources/static/js/ops-mgr/service-health/service-component-detail.html

```diff
@@ -42,7 +42,7 @@
                             <span class="column-title column-title-bottom">Message</span>
                         </div>
                         <div flex="20" layout="column" hide show-gt-md class="item-column md-list-item-text">
-                            <span class="item-title">{{alert.latestTimestamp | date : "MM/dd/yyyy 'at' h:mm"}}</span>
+                            <span class="item-title">{{'service-component-detail.time-locale' | translate:{date:alert.latestTimestamp} }} {{"service-component-detail.at" | translate}} {{'service-component-detail.time-locale2' | translate:{date:alert.latestTimestamp} }}</span>
```

Comment:
There are no English translations for `service-component-detail.at` and `service-component-detail.time-locale2`. Please fix this to only use `service-component-detail.time-locale`.